app/assets/javascripts/discourse/app/components/create-account.js

```diff
@@ -6,19 +6,19 @@ export default Component.extend({
   classNames: ["create-account-body"],
 
   userInputFocus(event) {
-    let label = event.target.parentElement.previousElementSibling;
-    if (!label.classList.contains("value-entered")) {
-      label.classList.toggle("value-entered");
+    let controls = event.target.parentElement;
```

Comment:
Does it matter that other user-field types still have the label outside of the `.controls` element? For example: 

https://github.com/discourse/discourse/blob/main/app/assets/javascripts/discourse/app/templates/components/user-fields/dropdown.hbs#L1 

I guess the focus in/out effect only applies to text inputs, is that correct?

Reply:
Yes this seems to be specific to user-field text inputs.